pkgs/development/python-modules/python3-otr/default.nix

```diff
@@ -0,0 +1,53 @@
+{ lib, fetchFromGitHub, buildPythonPackage, isPy3k, zope_interface
+, python3-application, cryptography, gmpy2 }:
+
+buildPythonPackage rec {
+  pname = "python3-otr";
+  version = "2.0.1";
+
+  disabled = !isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "AGProjects";
+    repo = "python3-otr";
+    rev = version;
+    sha256 = "sha256-XsJQIRZj04TkoWIddOc0evXwNFP/IYF/z/nO7mC0aVY=";
+  };
+
+  propagatedBuildInputs = [ python3-application zope_interface cryptography gmpy2 ];
+
+  # remove if https://github.com/AGProjects/python3-otr/pull/1 merged
+  patches = [ ./python3-otr-dep-name.patch ];
+
+  dontUseSetuptoolsCheck = true; # some tests fail with "TypeError: can't concat str to bytes"
```

Comment:
Adding a

```
  postPatch = ''
    substituteInPlace test.py --replace "'foobar" "b'foobar"
  '';
```

fixes it. And it's something you might like to send upstream because their tests are clearly broken.

This does rather raise questions about the quality of this project.